conf/base.config

```diff
@@ -36,6 +36,12 @@ process {
     time = { check_max( 20.h * task.attempt, 'time' ) }
     cache = 'lenient'
   }
+  withName:collect_stats {
+    cpus = { check_max( 1 * task.attempt, 'cpus' ) }
+    memory = { check_max( 200.GB * task.attempt, 'memory' ) }
```

Comment:
Does this process really need that much mem? It doesn't read the predictions at all.

Reply:
Probably not, but I would rather adjust/decrease the parameters afterwards.